jmp-application/src/main/resources/application.yml

```
@@ -0,0 +1,3 @@
+logging:
+  level:
+    root: ERROR
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)